docker/docker-compose.yml

```diff
@@ -105,6 +105,8 @@ services:
       - ./nats/:/etc/nats
     networks:
       - mainflux-base-net
+    ports:
```

Comment:
why this?

Reply:
This is not related to this particular PR, but it is nice to have NATS port opened so that you can run services locally for development purposes, and still be able to connect to the other services running in Docker.